src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { defineMessages, FormattedMessage } from 'react-intl';
-import { bigNumberify } from 'ethers/utils';
 
 import { TokenActivationPopover } from '~users/TokenActivation';
 import { Tooltip } from '~core/Popover';
-import { getFormattedTokenValue } from '~utils/tokens';
-import Numeral from '~core/Numeral';
-import { FullColonyFragment, UserLock, UserToken } from '~data/index';
+
+import { FullColonyFragment } from '~data/index';
 import { Address } from '~types/index';
 
 import styles from './UserTokenActivationButton.css';
+import UserTokenActivationDisplay from './UserTokenActivationDisplay';
+import { UserTokenBalanceData } from '~types/tokens';
```

Comment:
Same with these imports, they should be higher grouped with similar imports as them, and the CSS import last